internal/template/template.go

```diff
@@ -84,6 +88,12 @@ func WriteFile(in, out string, data map[string]interface{}) error {
 		return fmt.Errorf("parse template %q: %v", in, err)
 	}
 
+	// Rewrite if the file name has a template extension.
+	parts := strings.Split(out, ".")
```

Comment:
use filepath.Ext

Reply:
Done. Followed the suggestion here: https://stackoverflow.com/a/13027975

Reply:
I think that makes more sense if you want to trim the ext always. In this case since we know the exact suffix I think the best approach is strings.TrimSuffix as that will work just as well and does not create an if or need for any indexing.

Reply:
Oh I see what you mean. Done.